### PR TITLE
#167733635 Setup Test environment for the project

### PR DESCRIPTION
**Pull request by Baystef**
#### What does this PR do?
Setup unit testing to ensure all implementations are tested henceforth

#### Description of Task to be completed?
Setup mocha test runner, chai assertion library and have the `npm test` command working

#### How should this be manually tested?
After cloning the repo,
* `cd jubilee-backend`
* `npm i` to install essential dependencies
* Run `npm test` on the console while in the jubilee-backend directory

#### Any background context you want to provide?
Nil

#### What are the relevant pivotal tracker stories?
[#167733635](https://www.pivotaltracker.com/story/show/167733635)

#### Screenshots (if appropriate)
bn-initial-test

### CODE REVIEW

**Review by danielufeli — COMMENTED** [review]
Code looks good working well at my end

test

**Review by emmanuelnwankwo — APPROVED** [review]
Unit test is working

**Review by danielufeli — APPROVED** [review]
Code looks good working well at my end.

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "presets": [
+    "@babel/preset-env"
+  ]
+}
```
Comment (King-Success) [inline .babelrc]: This file needs a line break at the end

File: package.json [hunk 2]
```diff
@@ -1,30 +1,27 @@
...
+  "homepage": "https://github.com/andela/jubilee-backend#readme",
+  "devDependencies": {
+    "@babel/core": "^7.5.5",
+    "@babel/preset-env": "^7.5.5",
+    "@babel/register": "^7.5.5",
+    "chai": "^4.2.0",
+    "chai-http": "^4.3.0",
+    "mocha": "^6.2.0"
   }
 }
```
Comment (King-Success) [inline package.json:58]: Nice job fixing the package.json file

**Review by King-Success — APPROVED** [review]

**Review by King-Success — APPROVED** [review]

**Pull request merged**